src/main/java/com/googleinterns/zoomtube/transcriptParser/TranscriptParser.java

```diff
@@ -108,11 +109,17 @@ private void putTranscriptLinesInDatastore(Key lectureKey, Document document) {
       Node transcriptNode = transcriptNodes.item(nodeIndex);
       Element transcriptElement = (Element) transcriptNode;
       String lineContent = StringEscapeUtils.unescapeXml(transcriptNode.getTextContent());
-      Float lineStart = Float.parseFloat(transcriptElement.getAttribute(ATTR_START));
-      Float lineDuration = Float.parseFloat(transcriptElement.getAttribute(ATTR_DURATION));
-      Float lineEnd = lineStart.floatValue() + lineDuration.floatValue();
+
+      float lineStartSeconds = Float.parseFloat(transcriptElement.getAttribute(ATTR_START));
+      float lineDurationSeconds = Float.parseFloat(transcriptElement.getAttribute(ATTR_DURATION));
+      // I couldn't find any official way to convert a float seconds to long milliseconds without
```

Comment:
why not just store longs in transcriptElement?

This could be marked as a TODO.

Reply:
Was done in a previous PR, but the transcriptElement is parsed from the transcript API, which uses floats.